quickstart/www/assessment/standalone-assessment.php

```diff
@@ -0,0 +1,79 @@
+<?php
+    /**
+     * Copyright (c) 2017 Learnosity, MIT License
+     *
+     * Basic example of embedding a standalone assessment using Items API
+     * with `rendering_type: "assess"`.
+     */
+
+    // - - - - - - Section 1: server-side configuration - - - - - - //
+
+    // Include server side Learnosity SDK.
+    require_once __DIR__ . '/../../../src/vendor/autoload.php';
+    use LearnositySdk\Request\Init as LearnosityInit;
+    use LearnositySdk\Utils\Uuid as LearnosityUuid;
+
+    // Items API configuration parameters.
+    $request = [
+        'rendering_type' => 'assess',
+        'type'           => 'submit_practice',
+        'name'           => 'Items API Quickstart',
+        'state'          => 'initial',
+        'user_id'        => 'student_00012',    // Unique student identifier
+
+        // Reference of the Activity to retrieve from the Item bank.
+        // The Activity defines which Items will be served in this assessment.
+        'activity_template_id' => 'quickstart_examples_activity_template_001',
+
+        // Uniquely identifies this specific assessment attempt for save/resume,
+        // data retrieval and reporting purposes.
+        'session_id'     => LearnosityUuid::generate(),
```

Comment:
```suggestion
        'session_id'     => Uuid::generate(),
```

Reply:
Updated. I couldn't use your commit for some reason, so I replicated it manually.